<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Home</class>
 <widget class="QWidget" name="Home">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChatTime Home</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="widget_3" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>50</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>70</width>
                <height>50</height>
               </size>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="resources.qrc">:/images/plane.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="onlineNumber">
              <property name="font">
               <font>
                <pointsize>20</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Online Friends/Groupchat List</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QListWidget" name="friendList">
            <property name="font">
             <font>
              <pointsize>20</pointsize>
             </font>
            </property>
            <property name="cursor" stdset="0">
             <cursorShape>IBeamCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QListWidget {

	background-color: rgb(242, 233, 250);
    border: none;
    border-radius: 10px;

}
QListWidget::item {
    padding: 5px;
    border: 1px solid #D5D8DC;
    border-radius: 5px;
    margin: 2px 0;

}
QListWidget QScrollBar:vertical {
    width: 8px;
    background: #EAECEE;
}
QListWidget QScrollBar::handle:vertical {
    background: #BDC3C7;
    border-radius: 4px;
}
QListWidget::item:selected {
 
    color: black; 
}
</string>
            </property>
            <property name="iconSize">
             <size>
              <width>100</width>
              <height>100</height>
             </size>
            </property>
            <property name="viewMode">
             <enum>QListView::ListMode</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>710</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>710</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(242, 233, 250);

</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="resources.qrc">:/images/book.png</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="addFriendBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="toolTip">
              <string>add contact</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
        border: 2px solid #AED6F1;
        border-radius: 10px;  // Rounded corners
        background-color: #3498DB;  // Default color
        color: white;  // Text color
        font: bold 14px;  // Font and size
        padding: 8px;
    }
    QPushButton:hover {
        background-color: #5DADE2;  // Color when hovered
    }
    QPushButton:pressed {
        background-color: #2874A6;  // Color when pressed
    }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/images/add.svg</normaloff>:/images/add.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="startChatBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>50</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="toolTip">
              <string>start chat</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
        border: 2px solid #AED6F1;
        border-radius: 10px;  // Rounded corners
        background-color: #3498DB;  // Default color
        color: white;  // Text color
        font: bold 14px;  // Font and size
        padding: 8px;
    }
    QPushButton:hover {
        background-color: #5DADE2;  // Color when hovered
    }
    QPushButton:pressed {
        background-color: #2874A6;  // Color when pressed
    }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/images/startchat.svg</normaloff>:/images/startchat.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="groupChatBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
               <horstretch>50</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="toolTip">
              <string>start chat</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
        border: 2px solid #AED6F1;
        border-radius: 10px;  // Rounded corners
        background-color: #3498DB;  // Default color
        color: white;  // Text color
        font: bold 14px;  // Font and size
        padding: 8px;
    }
    QPushButton:hover {
        background-color: #5DADE2;  // Color when hovered
    }
    QPushButton:pressed {
        background-color: #2874A6;  // Color when pressed
    }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/images/groupchat.svg</normaloff>:/images/groupchat.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="AiChatStartBtn">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="toolTip">
              <string>ai chat</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
        border: 2px solid #AED6F1;
        border-radius: 10px;  // Rounded corners
        background-color: #3498DB;  // Default color
        color: white;  // Text color
        font: bold 14px;  // Font and size
        padding: 8px;
    }
    QPushButton:hover {
        background-color: #5DADE2;  // Color when hovered
    }
    QPushButton:pressed {
        background-color: #2874A6;  // Color when pressed
    }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/images/ai.svg</normaloff>:/images/ai.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="aipaint">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="toolTip">
              <string>ai chat</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
        border: 2px solid #AED6F1;
        border-radius: 10px;  // Rounded corners
        background-color: #3498DB;  // Default color
        color: white;  // Text color
        font: bold 14px;  // Font and size
        padding: 8px;
    }
    QPushButton:hover {
        background-color: #5DADE2;  // Color when hovered
    }
    QPushButton:pressed {
        background-color: #2874A6;  // Color when pressed
    }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/images/paint.svg</normaloff>:/images/paint.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="refreshBtn">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="toolTip">
              <string>refresh</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/images/refresh.svg</normaloff>:/images/refresh.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="logoutBtn">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>50</height>
              </size>
             </property>
             <property name="toolTip">
              <string>logout</string>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
        border: 2px solid #AED6F1;
        border-radius: 10px;  // Rounded corners
        background-color: #3498DB;  // Default color
        color: white;  // Text color
        font: bold 14px;  // Font and size
        padding: 8px;
    }
    QPushButton:hover {
        background-color: #5DADE2;  // Color when hovered
    }
    QPushButton:pressed {
        background-color: #2874A6;  // Color when pressed
    }</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="resources.qrc">
               <normaloff>:/images/logout.svg</normaloff>:/images/logout.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>35</width>
               <height>35</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
